layers/meta-opentrons/recipes-robot/nginx/files/nginx.conf

```diff
@@ -50,5 +50,21 @@ http {
             proxy_request_buffering  off;
             client_max_body_size     0;
         }
+
+        location /system {
+            proxy_http_version       1.1;
+            proxy_read_timeout       1h;
+            proxy_pass               http://127.0.0.1:32950;
+        }
+
+        # The /system/time endpoints live in the robot server, and thus
+        # the configuration here has to match the / config
+        location /system/time {
```

Comment:
same as the buildroot pr,
We should be able to nest location `/system/time` in the root config
everything else looks good though